Cargo.toml

```diff
@@ -7,3 +7,5 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+clap = "2.33.3"
```

Comment:
Clap v3's [derive macro is pretty cool](https://github.com/clap-rs/clap#using-derive-macros) if you wanted to try it out

Reply:
Noted, see 5eff499.